src/cvrp/constraints_validators.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module holds the various constraint validators used to configure the ECVRP
+
+@author: Cyril Obrecht
+@license: GPL-3
+@date: 2022-11-02
+@version: 0.1
+"""
+
+# CVRP
+# Copyright (C) 2022  A.Marie, K.Sonia, M.Jean, O.Cyril, V.Axel
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+from .individual import ConstraintValidator
+from .ecvrp import ECVRPSolution
+
+
+# pylint: disable=too-few-public-methods
+class BatteryTWValidator(ConstraintValidator[ECVRPSolution]):
+    """
+    Class in charge of checking if the battery capacity is never exeded
```

Comment:
Typo in the docstring. 
```suggestion
    Class in charge of checking if the battery capacity is never exceded
```